src/RoadmapTrack/RoadmapTrack.styled.tsx

```diff
@@ -0,0 +1,134 @@
+import styled, { css } from "styled-components";
+import { mediaQuery } from "styles/responsive";
+
+export const TrackContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+
+  ${mediaQuery.desktop} {
+    padding: 25px;
+    margin: -25px;
+    max-height: 100%;
+    overflow-y: auto;
+    &::-webkit-scrollbar {
+      display: none;
+    }
+    -ms-overflow-style: none; /* IE and Edge */
+    scrollbar-width: none; /* Firefox */
```

Comment:
why being explicit here? there are plugins that can do that for you on bundle 🙂

Reply:
What do you mean by "plugins on bundle"? If you are referring to `styled-components` usage of `stylis.js` for automatic vendor prefixing, it seems as though it's not a solution :(  I'm open to suggestions, but adding some additional dep just to remove 2 lines of CSS doesn't seem worth the bytes.

Reply:
Yeah, that was my thought 😓